release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -422,6 +431,15 @@ In the table, features are marked with the following statuses:
 AMD and Intel 64-bit architectures (x86-64-v2) will still be supported.
 ====
 
+[id="ocp-4-10-docker-config-location-deprecation"]
+==== Default Docker configuration location deprecation
+Previously, `oc` commands that used a registry configuration would obtain credentials from the Docker configuration location, which was `~/.docker/config.json` by default. This has been deprecated and will be replaced by a Podman configuration location in a future version of {product-title}.
+
+[id="ocp-4-10-empty-file-support-deprecation"]
+==== Empty file and stdout support deprecation in oc registry login
+
+Support for empty files using the `--registry-config` and `--to` flags in `oc registry login` has been deprecated. Support for `-` (or `stdout`) has also been deprecated as an argument when using `oc registry login`. They will be removed in a future version of {product-title}.
```

Comment:
```suggestion
Support for empty files using the `--registry-config` and `--to` flags in `oc registry login` has been deprecated. Support for `-` (stdout) has also been deprecated as an argument when using `oc registry login`. They will be removed in a future version of {product-title}.
```

probably better. stdout is just a shortcut for standard output. I suppose you can use that instead